### DISCUSSION TITLE
What is nodemon being used for here?

### DISCUSSION THREAD

**Discussion by thgh** [payloadcms/public-demo#20] [Q&A]

**Comment by jacobsfletch** [comment]
@thgh I am going to move this issue to a discussion so we can discuss further 👍

**Comment by jacobsfletch** [comment]
Ok so `nodemon` is used to recompile server files when they change. The Payload server needs to restart when the config changes. We use `nodemon` to watch all `.ts` files. This is the standard setup for all Payload projects, and is what comes with a fresh `npx create-payload-app`.

**Reply by jacobsfletch** [reply]
Ahh I'm now seeing your related PR here: https://github.com/payloadcms/public-demo/pull/16
